content/articles/core-concepts-of-django-to-master/index.md

```diff
@@ -0,0 +1,166 @@
+<!-- Author Biography -->
+Name: ARAFAT OLAWUMI OLAYIWOLA
+I am Software and Associate Cloud Engineer who is passionate about the over-whelming technologies out there. I develop a scalable and user friendly applications that run on all platforms and responsive. I love Django like my girlfriend. Undergraduate of Computer Engineering here in Nigeria.
+
+<!-- My Picture -->
+![Arafat Olayiwola](/core-concepts-of-django-for-mastering/Arafat-Olayiwola.jpg)
+
+![django-img](/core-concepts-of-django-for-mastering/django-image.png)
+
+<!-- INTRODUCTION -->
+Are you  a beginner or an intermediate of Django? Join me to explore more on this superb and sweet framework used to setup web apps by connecting to server. You don't want to miss this!!
```

Comment:
This sentence is not necessary, The reader already knows why they are here after being lured by the title